src/js/franklin-dashboard.js

```diff
@@ -7,102 +7,117 @@ import 'angular-ui-router';
 import 'angular-animate';
 import 'angular-toastr';
 import 'satellizer';
-// require('../../node_modules/foundation-apps/js/angular/foundation');
+//import 'angular-foundation';
+//import 'angular-foundation/mm-foundation';
+
 
 /* Franklin Dashboard modules */
-require('./config/ngConstants');
+import './config';
 import './services';
-import { DashboardComponent } from './dashboard/dashboard.controller';
-import { LoginComponent } from './login/login.controller';
+import {
+  DashboardComponent
+}
+from './dashboard/dashboard.controller';
+import {
+  LoginComponent
+}
+from './login/login.controller';
 
 
 angular
-  .module('franklin-dashboard',
-    [
-      'ngAnimate',
-      'toastr',
-      'satellizer',
-      'franklin-dashboard.config',
-      'ui.router',
-      'franklin-dashboard.services'
-    ]
-  )
+  .module('franklin-dashboard', [
+    'ngAnimate',
+    'toastr',
+    'satellizer',
+    'franklin-dashboard.config',
+    'ui.router',
+    'franklin-dashboard.services'//,
+    //'mm.foundation'
+  ])
   .constant('VERSION', packageJson.version)
-  .config(
-    ($authProvider, ENV, $stateProvider, $urlRouterProvider, toastrConfig) => {
-
-      //angular routing depending on login status
-      $stateProvider
-        .state('logged', {
-          url: '/dashboard',
-          templateUrl: 'dashboard/dashboard.html',
-          resolve: {
-            loginRequired: loginRequired
-          }
-        })
-        .state('logout', {
-          url: '/login',
-          templateUrl: 'login/login.html',
-          resolve: {
-            skipIfLoggedIn: skipIfLoggedIn
-          }
-        });
-
-      $urlRouterProvider.otherwise('/dashboard');
-
-      //Github login configuration
-      $authProvider.withCredentials = false;
-
-      $authProvider.github({
-        clientId: ENV.GITHUB_CLIENT_ID,
-        url: ENV.FRANKLIN_API_URL + '/auth/github/'
-      });
+  .config(($authProvider, ENV, $stateProvider, $urlRouterProvider, toastrConfig, $resourceProvider, $interpolateProvider) => {
 
+    $interpolateProvider.startSymbol('{[{').endSymbol('}]}');
```

Comment:
Right I get why you're changing the templating symbols we do it all the time what I was asking was why you were using ``` {[{ }]}``` instead of ```[[ ]]``` which we already use.